extensions/antd-open-browser/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "antd-open-browser",
+  "title": "antd-open-browser",
+  "description": "antd-open-browser",
+  "icon": "logo.png",
+  "author": "crazyair",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "antd",
+      "subtitle": "快速打开 antd 组件",
+      "description": "快速打开 antd 组件",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
fix